### ISSUE TITLE
can't operate servos Ubuntu 20.04

### ISSUE THREAD

**Issue by siyengar04** [kdkalvik/pololu_maestro_ros#1]
does this library still work? I can't seem to control the servos with a Docker container running Ubuntu 20.04. `lsusb` still brings up the Pololu Maestro device, but none of the commands work. I have servos on channels 0-2 and none of them seem to want to move.

**Comment by itskalvik** [comment]
No, I am not actively maintaining it anymore. Perhaps try it on a barebone Ubuntu machine instead of a docker image
